spec/index.bs

```diff
@@ -4049,6 +4076,22 @@ enum GPUStencilOperation {
 };
 </script>
 
+<div algorithm>
+    <dfn abstract-op>validating GPUDepthStencilStateDescriptor</dfn>(descriptor)
```

Comment:
I think we wanted to move to a style where algorithm names have verb tense like "validate GPUDepthStencilStateDescriptor"

Reply:
Yes, sure, but let's do it everywhere consistently as a separate change.
So far, all the algorithms are named "validating ..."